app/src/main/java/io/intrepid/contest/screens/entrysubmission/join/JoinPresenter.java

```diff
@@ -4,11 +4,16 @@
 
 import io.intrepid.contest.base.BasePresenter;
 import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.InvitationResponse;
+import io.reactivex.disposables.Disposable;
 
 class JoinPresenter extends BasePresenter<JoinContract.View> implements JoinContract.Presenter {
 
     JoinPresenter(@NonNull JoinContract.View view, @NonNull PresenterConfiguration configuration) {
         super(view, configuration);
+
+        // todo: set the authentication token when the app is downloaded, not here
+        userSettings.setAuthenticationToken("503adec2-6b60-422d-9f94-3095af9c1416");
```

Comment:
I will move it to the `SplashPresenter`, and any other class that uses it can do it through `userSettings.getAuthenticationToken()`. I don't think the value itself needs to be somewhere else though, since it's just to test the api calls while we don't implement the "create user" story.